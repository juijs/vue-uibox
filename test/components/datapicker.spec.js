import { mount } from 'vue-test-utils'
import UixDatepicker from '@/components/datepicker'

describe('datepicker.js', () => {
    const wrapper = mount(UixDatepicker, {
        propsData: {
            dateFormat: 'yyyy/MM/dd',
            date: "2018/09/01",
        }
    });

    it('snapshot!!', () => {
        wrapper.vm.$nextTick(() => {
            expect(wrapper.vm.$el).toMatchSnapshot();
        });
    });
})