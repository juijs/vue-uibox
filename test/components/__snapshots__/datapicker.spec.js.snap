// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`datepicker.js snapshot!! 1`] = `
<div
  class="datepicker"
>
  <div
    class="head"
  >
    <div
      class="prev"
    >
      <i
        class="icon-chevron-left"
      />
    </div>
     
    <div
      class="title"
    >
      2018.09
    </div>
     
    <div
      class="next"
    >
      <i
        class="icon-chevron-right"
      />
    </div>
  </div>
   
  <table
    class="body"
  >
    <thead>
      <tr>
        <th>
          SU
        </th>
        <th>
          MO
        </th>
        <th>
          TU
        </th>
        <th>
          WE
        </th>
        <th>
          TH
        </th>
        <th>
          FR
        </th>
        <th>
          SA
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td
          class="none"
        >
          26
        </td>
        <td
          class="none"
        >
          27
        </td>
        <td
          class="none"
        >
          28
        </td>
        <td
          class="none"
        >
          29
        </td>
        <td
          class="none"
        >
          30
        </td>
        <td
          class="none"
        >
          31
        </td>
        <td
          class="active"
        >
          1
        </td>
      </tr>
      <tr>
        <td>
          2
        </td>
        <td>
          3
        </td>
        <td>
          4
        </td>
        <td>
          5
        </td>
        <td>
          6
        </td>
        <td>
          7
        </td>
        <td>
          8
        </td>
      </tr>
      <tr>
        <td>
          9
        </td>
        <td>
          10
        </td>
        <td>
          11
        </td>
        <td>
          12
        </td>
        <td>
          13
        </td>
        <td>
          14
        </td>
        <td>
          15
        </td>
      </tr>
      <tr>
        <td>
          16
        </td>
        <td>
          17
        </td>
        <td>
          18
        </td>
        <td>
          19
        </td>
        <td>
          20
        </td>
        <td>
          21
        </td>
        <td>
          22
        </td>
      </tr>
      <tr>
        <td>
          23
        </td>
        <td>
          24
        </td>
        <td>
          25
        </td>
        <td>
          26
        </td>
        <td>
          27
        </td>
        <td>
          28
        </td>
        <td>
          29
        </td>
      </tr>
      <tr>
        <td>
          30
        </td>
        <td
          class="none"
        >
          1
        </td>
        <td
          class="none"
        >
          2
        </td>
        <td
          class="none"
        >
          3
        </td>
        <td
          class="none"
        >
          4
        </td>
        <td
          class="none"
        >
          5
        </td>
        <td
          class="none"
        >
          6
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
